packages/astro/src/default/env-default.d.ts

```diff
@@ -9,8 +9,9 @@ interface WebContainerConfig {
 
 declare module 'tutorialkit:override-components' {
   const topBar: typeof import('./src/default/components/TopBar.astro').default;
+  const dialog: typeof import('@tutorialkit/react/dialog').default;
 
-  export { topBar as TopBar };
+  export { topBar as TopBar, dialog as Dialog };
```

Comment:
```suggestion
  const TopBar: typeof import('./src/default/components/TopBar.astro').default;
  const Dialog: typeof import('@tutorialkit/react/dialog').default;

  export { TopBar, Dialog };
```

Reply:
The lowercase ones are due to ESLint rules:

```
Variable name `TopBar` must match one of the following formats: camelCase, UPPER_CASEeslint[@typescript-eslint/naming-convention](https://typescript-eslint.io/rules/naming-convention)
```

Reply:
Oh LUL, let's keep them then :+1: